example/spec/loadTest/runScaledTest.js

```diff
@@ -0,0 +1,362 @@
+/* eslint-disable no-await-in-loop */
+
+'use strict';
+
+const path = require('path');
+const fs = require('fs');
+const pWaitFor = require('p-wait-for');
+
+const { getAggregateMetricQuery, getInvocationCount } = require('@cumulus/integration-tests/metrics');
+const { generateIterableTestDirectories } = require('@cumulus/integration-tests/utils');
+const { LambdaStep } = require('@cumulus/integration-tests/sfnStep');
+const { listS3ObjectsV2 } = require('@cumulus/aws-client/S3');
+const { providers, collections, granules } = require('@cumulus/api-client');
```

Comment:
Nit, but you can sort by alphabetical order here 😀

Reply:
Oooo!   I like that one.

Reply:
227f01512